Deployables/deployables_subsystem.py

```diff
@@ -0,0 +1,229 @@
+"""This python program represents a simulated version of the Deployables (GPIOS) subsystem 
+
+Each deployable is associated with a burwire pin, and a single feedback switch pin that indicates 
+whether the deployable has been successfully deployed. Each Pin represents a GPIO on the OBC.
+
+Burnwire GPIOS are OUTPUTS
+Feedback Switch GPIOS are INPUTS
+
+The Fprime component will read these simulated feeback switches state (HIGH or LOW).
+Timers activated upon a burnwire set HIGH simulate the feedback switch state changing.
+The Fprime component will be responsible for keeping track of the state of the deployables.
+
+Switches are assumed to be normally open, meaning: 
+    A switch pin reading HIGH indicates that the deployable has been successfully deployed.
+
+For now the sub system communicates with the OBC using strings over a TCP socket.
+
+Until we know more system specs I am assuming there are 2 types of commands that can be sent: 
+    - Burwire Set - Set a GPIO pin HIGH or LOW (toggle a burnwire)
+    - Switch Request - Request a GPIO value (get feedback switch state)
+
+# Example setting the DFGM burnwire active 
+    BURNWIRE_SET_COMMAND:DFGM:1
+
+# Example requesting the state of the DFGM feedback switch
+    SWITCH_REQUEST_COMMAND:DFGM
+
+For now you can test your commands using netcat (nc) from the command line, and piping the command 
+to the socket from a seperate text file. I have also added a brief bash script to test this program.
+
+Usage: deployables_subsystem.py 
+"""
+
+import sys
+import threading
+import random
+
+sys.path.append('..') # Add parent directory to path so we can import modules from there
+import socket_stuff # pylint: disable=C0413
+import command_handler # pylint: disable=C0413
+
+
+DEFAULT_HOST = '127.0.0.1'
+DEFAULT_PORT = 1811
+
+
+BURNWIRE_SET_COMMAND = "burnwire_set"
+SWITCH_REQUEST_COMMAND = "switch_request"
+
+BURNWIRE_PIN = "burnwire_pin"
```

Comment:
Hmm, I guess Python doesn't really have the concept of a struct, although an object/class has a lot of similar properties. Python also doesn't really have #defines (or constants?). The two shortcomings are working together to make your C like dictionary, um, unique ;->

Reply:
Good point. Regarding the C style mock '#defines' : Being less experienced with python I realized after writing this that it's not very 'pythonic'.  In this document it mentions that 'user defined constants' are literally just the variable being spelt a certain format (all caps w/ underscore in between):  https://realpython.com/python-constants/  . Also after making the  deployables_state dictionary I thought about using a class/object style, but it seemed like a dictionary was sufficient for now. I now think for something like the EPS or ADCS which is more 'intelligent' and has more functionality, it would probably make more sense to represent the state as an object.